src/Uno.UWP/Storage/StorageFile.native.wasm.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage.FileProperties;
+using Windows.Storage.Streams;
+using SystemPath = global::System.IO.Path;
+
+namespace Windows.Storage
+{
+	public partial class StorageFile
+	{
+		internal static StorageFile GetFileFromNativePathAsync(Guid guid, string name, string contentType) => new StorageFile(new NativeFileSystem(guid, name, contentType));
+
+		private sealed class NativeFileSystem : ImplementationBase
```

Comment:
What about this (same for the javascript one) ? 

```suggestion
		private sealed class NativeStorageFile : ImplementationBase
```